src/shared/sam/localLambdaRunner.ts

```diff
@@ -248,13 +242,6 @@ export async function invokeLambdaFunction(
             runtime: config.runtime as Runtime,
             debug: !config.noDebug,
         })
-        if (config.outFilePath) {
```

Comment:
@justinmk3 outFilePath seems to not be used anywhere else and had no comment on what it does, please comment if that is not correct (and add a doc comment :D)

Reply:
`outFilePath` was a hack for the old python codepath. Not needed since 4471ffd7fe5d105c014a7581cc9cbb67bac36465 , good to remove it!